slices/openssh-sftp-server.yaml

```diff
@@ -0,0 +1,15 @@
+package: openssh-sftp-server
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - openssh-client_bins
+    contents:
+      /usr/lib/openssh/sftp-server:
+      /usr/lib/sftp-server:
+
+  standard:
+    essential:
+      - openssh-client_standard
```

Comment:
Same question here as above - if I install the `bins` slice alone, I won't get `openssh-client_standard`, so would those 2 bins be functional?

Reply:
Refactored the standard slices, should work now